charts/karpenter/templates/_helpers.tpl

```diff
@@ -0,0 +1,68 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
Is it necessary to have this implementation? Can you briefly outline the features that this gives us? I worry that it will be challenging to maintain and perhaps unnecessary complexity.